taskt/UI/Forms/ScriptBuilder Forms/frmScriptBuilder.cs

```diff
@@ -562,6 +562,9 @@ public void CopyTreeViewNodes(TreeNode copiedTreeNode, TreeNode originalTreeNode
 
         private void txtCommandSearch_TextChanged(object sender, EventArgs e)
         {
+            if (txtCommandSearch.Text == "Type Here to Search")
```

Comment:
Please assign this text to a private field and use that field reference wherever required in this file.

Reply:
done